spirv_cross/src/bindings.rs

```diff
@@ -8,7 +8,7 @@ pub mod root {
         #[allow(unused_imports)]
         use self::super::super::root;
         pub type Id = ::std::os::raw::c_uint;
-        #[repr(i32)]
+        #[repr(u32)]
```

Comment:
Any idea why this happened with bindgen? Seems a bit strange for it to switch like this between compilers, but I guess either `u32`/`i32` shouldn't make a difference here. I wonder if we can set it as a bindgen setting somewhere.